src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int number = new Random().nextInt(100);
+        return new Ball(new ColorSupplier().getRandomColor(), number);
```

Comment:
Overloaded expression, please introduce some variables. Also creation of Random and ColorSupplier objects should be in class level variables